en/docs/architecture-diagram/architecture-diagram.md

```diff
@@ -69,62 +62,77 @@ isolated function getSupportedCurrencies() returns string[]|error {
+
+1. To set up a Ballerina project as a VS Code workspace, add all the Ballerina packages to the workspace using one of the following methods:
+
+      -  Use the **VS Code workspace file** from an already created workspace to import its packages to a new workspace.
+
+         > **Info:** The workspace file of the [Google Cloud Platform demo](https://github.com/ballerina-guides/gcp-microservices-demo/tree/project-design-demo) project is available in the repository.
+
+      -  Add each Ballerina package to the workspace using the **Add folder to workspace** option in the VS Code editor.
 
-The [Google Cloud Platform demo](https://github.com/ballerina-guides/gcp-microservices-demo/tree/project-design-demo) is used to demonstrate the different capabilities of the diagram. This sample has 10 services (9 GRPC and one HTTP) in 10 separate Ballerina packages. The `ui folder` is not a Ballerina package, hence would not make an impact.
+      When the workspace is set up, you will see the following folder structure in the **Explorer**:
 
-As per the [Related concepts](#a-project-workspace) section, each of these packages needs to be loaded into a VS Code workspace. This could be done using any of the following approaches.
-1. Use the VS Code workspace file that is available in the repository
-2. Add each Ballerina package to the workspace using the `Add folder to workspace` feature available in VS Code
+      >**Note:** The workspace name may be different if you set up the workspace from scratch.
 
-If the workspace has been set up accurately, you would be able to see the following folder structure in the explorer.
->**Note:** The workspace name would be different if you set up the workspace from scratch.
+      <img src="../../img/pdd-workspace-structure.png" alt="Workspace Structure" width="400"/>
 
-![Workspace Structure](../img/pdd-workspace-structure.png)
+2. Once the packages are added to the workspace, activate the Ballerina Design Diagram generator via the `Ballerina: Project Design` command in the command palette as shown below.
 
-Once the workspace has been set up, the Ballerina design diagram generator can be activated via the `Ballerina: Project Design` command. This command can be found in the VS Code command palette as shown below.
-![Extension Command](../img/pdd-extension-command.gif)
+      ![Extension Command](../img/pdd-extension-command.gif)
 
-The execution of this command will open up a VS Code webview panel that will contain three types of design diagrams. You can navigate between these diagrams using the side panel available on the diagram webview.
+## Explore the Project Design view
 
-### Service Interaction Diagram
+Let's explore the features of the Design Diagram using the [Google Cloud Platform](https://github.com/ballerina-guides/gcp-microservices-demo/tree/project-design-demo) Ballerina project that you set up in the previous topic.
```

Comment:
I couldn't find a reference to this in the UI and I thought the Service Diagram 1, Service Diagram 2, and Type Diagram was it (according to the UI).

Please correct me if I am wrong.